upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -105,6 +106,15 @@ func (tf *TemplateFunctions) AddTo(dest template.FuncMap) {
 	}
 
 	dest["ProxyEnv"] = tf.ProxyEnv
+
+	if tf.cluster.Spec.Networking != nil && tf.cluster.Spec.Networking.Flannel != nil {
+		flannelBackendType := tf.cluster.Spec.Networking.Flannel.Backend
+		if flannelBackendType == "" {
+			glog.Warningf("Defaulting flannel backend to udp (not a recommended configuration)")
```

Comment:
Should we tell the user what to do?  You need to set this, or you need to migrate to this?  We are warning them, but not helping them on what to do.

Reply:
Upgrade path I followed today on a running cluster. Be aware that this creates downtime/connectivity issues ~1 minute. Steps followed:

1. `kops edit cluster` <- change udp for vxlan
2. `kubectl edit configmap -n kube-system kube-flannel-cfg` <- change udp for vxlan
3. `kubectl delete pod -n kube-system -l app=flannel`